Mage.Sets/src/mage/cards/w/WowzerTheAspirational.java

```diff
@@ -0,0 +1,92 @@
+package mage.cards.w;
+
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.common.AttacksTriggeredAbility;
+import mage.abilities.condition.CompoundCondition;
+import mage.abilities.condition.Condition;
+import mage.abilities.condition.IntCompareCondition;
+import mage.abilities.condition.common.CitysBlessingCondition;
+import mage.abilities.condition.common.HaveInitiativeCondition;
+import mage.abilities.condition.common.MonarchIsSourceControllerCondition;
+import mage.abilities.condition.common.PermanentsOnTheBattlefieldCondition;
+import mage.abilities.decorator.ConditionalInterveningIfTriggeredAbility;
+import mage.abilities.dynamicvalue.common.CountersControllerCount;
+import mage.abilities.effects.common.WinGameSourceControllerEffect;
+import mage.abilities.hint.ConditionHint;
+import mage.abilities.hint.Hint;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.ComparisonType;
+import mage.constants.SubType;
+import mage.constants.SuperType;
+import mage.counters.CounterType;
+import mage.filter.common.FilterControlledPermanent;
+import mage.game.Game;
+
+import java.util.UUID;
+
+/**
+ * @author Cguy7777
+ */
+public class WowzerTheAspirational extends CardImpl {
+
+    private static final Condition condition = new CompoundCondition(
+            new WowzerTheAspirationalCondition(),
+            new PermanentsOnTheBattlefieldCondition(new FilterControlledPermanent(SubType.BLOOD)),
+            new PermanentsOnTheBattlefieldCondition(new FilterControlledPermanent(SubType.CLUE)),
+            new PermanentsOnTheBattlefieldCondition(new FilterControlledPermanent(SubType.FOOD)),
+            new PermanentsOnTheBattlefieldCondition(new FilterControlledPermanent(SubType.MAP)),
+            new PermanentsOnTheBattlefieldCondition(new FilterControlledPermanent(SubType.POWERSTONE)),
+            new PermanentsOnTheBattlefieldCondition(new FilterControlledPermanent(SubType.TREASURE)),
+            MonarchIsSourceControllerCondition.instance,
+            CitysBlessingCondition.instance,
+            HaveInitiativeCondition.instance);
+    private static final Hint hint = new ConditionHint(
```

Comment:
the problem with the single hint is that it doesn't tell you what you're missing

but I don't have a strong opinion here really, if you like the way you have it, just add to SKIP_LIST_MISSING_ABILITIES in the verify test

Reply:
After giving it a try, I agree that having individual hints is much better. Thank you. :-)